lib/translate/common.cc

```diff
@@ -31,8 +29,91 @@ void DebugLog(const char* format, ...) {
   va_end(args);
 }
 
-void SetDebugCallback(void (*callback)(const char*)) {
+void SetDebugCallback(void (*callback)(const char *)) {
   debug_callback = callback;
 }
 
-}  // namespace translator
+StatusCode MakeScsiOpcode(uint8_t val, scsi_defs::OpCode &opcode) {
+  if (val > 0xaf) {
```

Comment:
It's also not just the opcodes with val > 0xaf that we don't support. There are lots of gaps in the opcodes that we support so maybe we should just call `ScsiOpcodeToString` and see whether that returns `"INVALID_OPCODE"`

Reply:
ok how about `if dynamic_cast<> != null`

Reply:
i've researched a bit and yeah LUTs are the way to go apparently.